<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorDialog</class>
 <widget class="QDialog" name="CalculatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计算器</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="m_editx">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_label">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_edity">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_editz">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
